frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -64,12 +83,59 @@ function Application(): JSX.Element {
 		[handleSetTimeStamp],
 	);
 
-	const { topOperations, serviceOverview, topLevelOperations } = useSelector<
-		AppState,
-		MetricReducer
-	>((state) => state.metrics);
+	const serviceOverviewQuery = {
```

Comment:
also no need of this object let use this one directly

Reply:
Okay